framework/src/mfem/bcs/MFEMRWTE10IntegratedBC.C

```diff
@@ -41,21 +41,14 @@ MFEMRWTE10IntegratedBC::MFEMRWTE10IntegratedBC(const InputParameters & parameter
     _mu(getParam<Real>("mu")),
     _epsilon(getParam<Real>("epsilon")),
     _omega(2 * M_PI * getParam<Real>("frequency")),
-    _a1_vec(to3DMFEMVector(getParam<RealVectorValue>("port_length_vector"))),
-    _a2_vec(to3DMFEMVector(getParam<RealVectorValue>("port_width_vector"))),
-    _a3_vec(CrossProduct(_a1_vec, _a2_vec)),
-    _a2xa3(CrossProduct(_a2_vec, _a3_vec)),
-    _a3xa1(CrossProduct(_a3_vec, _a1_vec)),
-    _v(mfem::InnerProduct(_a1_vec, _a2xa3)),
-    _kc(M_PI / _a1_vec.Norml2()),
+    _a1_vec(const_cast<mfem::real_t *>(&getParam<RealVectorValue>("port_length_vector")(0)), 3),
+    _a2_vec(const_cast<mfem::real_t *>(&getParam<RealVectorValue>("port_width_vector")(0)), 3),
```

Comment:
```suggestion
    _a1_vec(const_cast<mfem::real_t *>(&getParam<RealVectorValue>("port_length_vector")(0)), Moose::dim),
    _a2_vec(const_cast<mfem::real_t *>(&getParam<RealVectorValue>("port_width_vector")(0)), Moose::dim),
```

Reply:
If we removed that static assertion, I'm assuming you'd like your code to still work, slash, I'd like to minimize the number of things I'd have to fix in the MOOSE herd